docs/sources/flow/getting-started/collect-opentelemetry-data.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Collect OpenTelemetry data
+weight: 300
+---
+
+# Collect OpenTelemetry data
+
+Grafana Agent Flow can be configured to collect [OpenTelemetry][] data and
```

Comment:
s/OpenTelemetry/OpenTelemetry Protocol/

(or OTLP)

Reply:
This applies to other places in this doc: always when referring to the data spec, please use OTLP.

Reply:
Thanks! Will fix 👍